GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -0,0 +1,184 @@
+        * Review devops pipeline YAML and only keep templates relevant to your enterprise patterns.
+            * Remove jobName configurations not relevant to your enterprise patterns.
+                ```yaml
+                configurationMatrix:
+                - jobName: az_service_single_region
+                terraformTemplatePath: 'infra/templates/az-service-single-region'
+                terraformWorkspacePrefix: 'sr'
+                environmentsToTeardownAfterRelease:
+                - 'devint'
+                ```
+            > NOTE: If new to Cobalt, we recommend keeping the az_hello_world template as a starter template. This step can also be completed later as a code commit to your repo.
+        * Save and run
+            > NOTE: Azure Devops forces a run so expect this to fail. Future steps will solve this problem.
+
+2. Provision Azure resources needed for Azure Devops pipeline
+
+    * Create a registered Azure AD (AAD) app for Cobalt deployments
+        * Sign-in to your organization's Azure account. (https://portal.azure.com)
+        * Filter for Azure Active Directory and navigate to it's menu
+        * Select App registrations from the menu blade
+        * Click [Add/+] New registration then enter a name for the application (ex. cobalt-hw-admin-sp)
+        * Choose single tenant as a supported account type to keep things simple.
+        * Click Register
+
+    * Configure the new AAD app as a Cobalt admin service-principal/service-endpoint
+        * From the App registrations service blade, select the Certificates & Secrets tab
+        * Click [+New client secret] from within Client secrets menu then enter a description (ex. rbac)
+            ![image](https://user-images.githubusercontent.com/10041279/63461963-69d35a80-c41f-11e9-8d4a-c72235177fb3.png)
+        * Click Add
+            > NOTE: Take note of the generated client secret (only displayed once). This will be used for your Azure Devops Service Connection in step 3.
+            > Important: Secrets that lead with special characters may cause errors. (ex.!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~)
+        * From the App registrations service blade, select Overview.
+            > NOTE: Take note of the Application (client) ID. This will also be used for your Azure Devops Service Connection in step 3.
+
+    * Grant newly created Service Principal a Contributor role to your preferred enterprise subscription
+        * Filter for subscriptions and navigate to the subscriptions list
+        * Either choose a subscription or create a new one
+        * Select your chosen subscription then select the Access control (IAM) tab from the menu blade.
+        * Click [+/Add] and select Add role assignment
+            * From the sub-menu, select 'Contributor' as a role from the drop down and search for the newly created Service Principal (i.e. cobalt-hw-admin-sp)
```

Comment:
This needs to be 'Owner'

Reply:
Doc updated, but image still shows Contributor. Consider removing the image (or replacing).